src/circuit/gadget/ecc/chip/mul/complete.rs

```diff
@@ -58,9 +59,8 @@ impl Config {
 
                 // k_{i} = z_{i} - 2⋅z_{i+1}
                 let k = z_next - Expression::Constant(pallas::Base::from_u64(2)) * z_prev;
-                let k_minus_one = k.clone() - Expression::Constant(pallas::Base::one());
                 // (k_i) ⋅ (k_i - 1) = 0
```

Comment:
```suggestion
                // (k_i) ⋅ (1 - k_i) = 0
```
Or replace this comment entirely with something about it being a boolean constraint.